cmds/ip/ip.go

```diff
@@ -146,14 +210,19 @@ func linkset() {
 }
 
 func link() {
+	if len(arg) == 1 {
+		linkshow()
+		return
+	}
+
 	cursor++
 	whatIWant = "show|set"
 	cmd := arg[cursor]
 
 	switch cmd {
-	case "show":
+	case "show", "sho", "sh":
```

Comment:
I see this code repeated. I wonder if it makes sense to have a function like such as:
matches(s, command string, min int) {
see if s matches shorter and shorter substrings, but at least int chars
}